api.md

```diff
@@ -1307,3 +1398,5 @@ and then inspect `dump_file.txt` before sending it together with a bug report or
 [adaptive-chunking]: /using-timescaledb/adaptive-chunking
 [memory-units]: https://www.postgresql.org/docs/current/static/config-setting.html#CONFIG-SETTING-NAMES-VALUES
 [telemetry]: /using-timescaledb/telemetry
+[drop-chunks]: #drop_chunks
```

Comment:
the link refs above use underscores vs. hyphens.

Reply:
Note entirely sure what format you want, but i updated to match the chunk relation size formatting.